tests/staticfiles_tests/test_checks.py

```diff
@@ -0,0 +1,89 @@
+from __future__ import unicode_literals
```

Comment:
Please rebase your branch. Master now supports Python 3 only, so `unicode_literals` may be removed.